source/geometries/Next100SiPM.cc

```diff
@@ -71,6 +76,34 @@ void Next100SiPM::Construct()
 
   GeometryBase::SetLogicalVolume(sipm_logic_vol);
 
+  // COATING /////////////////////////////////////////////
+
+  G4LogicalVolume* coating_logic_vol;
```

Comment:
Is there any reason to declare this logical volume out of the `if` clause?

Reply:
I did it in the same manner it's declared in GenericPhotosensors. I thought it was a matter of convention.